libraries/api-documenter/src/markdown/MarkdownGenerator.ts

```diff
@@ -21,48 +22,49 @@ import {
   MarkupStructuredElement
 } from '@microsoft/api-extractor';
 
-import { ApiJsonFile } from './ApiJsonFile';
-import { BasePageRenderer } from './BasePageRenderer';
-import { RenderingHelpers } from './RenderingHelpers';
-import { MarkupBuilder } from './MarkupBuilder';
-import { DocumentationNode } from './DocumentationNode';
+import { DocItemSet, DocItem, DocItemKind, IDocItemSetResolveResult } from '../DocItemSet';
+import { RenderingHelpers } from '../RenderingHelpers';
+import { MarkupBuilder } from '../MarkupBuilder';
+import { MarkdownRenderer, IMarkdownRenderApiLinkArgs } from '../MarkdownRenderer';
 
 /**
- * This is the main engine that reads *.api.json input files and generates IMarkupPage data structures,
- * which are then rendered using an BasePageRenderer subclass.
+ * Renders API documentation in the Markdown file format.
+ * For more info:  https://en.wikipedia.org/wiki/Markdown
  */
-export class Documenter {
-  private readonly _apiJsonFiles: ApiJsonFile[] = [];
+export class MarkdownGenerator {
+  private _docItemSet: DocItemSet;
+  private _outputFolder: string;
 
-  public loadApiJsonFile(apiJsonFilePath: string): void {
-    this._apiJsonFiles.push(ApiJsonFile.loadFromFile(apiJsonFilePath));
+  public constructor(docItemSet: DocItemSet) {
+    this._docItemSet = docItemSet;
   }
 
-  public writeDocs(renderer: BasePageRenderer): void {
-    console.log(os.EOL + `Deleting old *${renderer.outputFileExtension} files...` + os.EOL);
-    renderer.deleteOutputFiles();
+  public generateFiles(outputFolder: string): void {
+    this._outputFolder = outputFolder;
 
-    for (const apiJsonFile of this._apiJsonFiles) {
-      this._writePackagePage(apiJsonFile, renderer);
+    console.log();
```

Comment:
>console.log(); [](start = 4, length = 14)

keep this?

Reply:
This skips a line in the console output

---
In reply to: [141506540](https://github.com/Microsoft/web-build-tools/pull/381#discussion_r141506540) [](ancestors = 141506540)